### ISSUE TITLE
Solution (?)

### ISSUE THREAD

**Issue by coadybyrne** [coadybyrne/plan-a-website#1]
![berried in berries](https://user-images.githubusercontent.com/97690750/157807304-de3abd4a-2e2c-4444-990b-f278b00c7e0c.png)
So I think this is right. At the very least, it's what I used to plan the site I submitted to MarkBot, and that went through fine, hahah.
The only thing it's missing is the Homepage-Berries-Resources navigation that's on every page, but I wasn't sure how to draw that in without making a huge mess.
@sharmahAC

**Comment by sharmahAC** [comment]
This is not wrong but we always use left to right approach to show hierarchy when planning to code a website.
Like Home – Berries - Resources

From: Coady Byrne ***@***.***>
Date: Friday, March 11, 2022 at 12:26 AM
To: coadybyrne/plan-a-website ***@***.***>
Cc: Harsh Sharma ***@***.***>, Mention ***@***.***>
Subject: [coadybyrne/plan-a-website] Solution (?) (Issue #1)

AC ITS Caution: This email originated from an external sender. Be careful of phishing attacks.

Security Tip of the Month: Report to the ITS Cyber Security Unit suspicious emails using the “Report Phishing” fish icon, found on the upper right side of your Microsoft Outlook toolbar ribbon. By doing so, you will securely delete suspicious messages from your inbox. If you do not have this fish icon, please contact the ITS Service Desk to install it.


[berried in berries]<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F97690750%2F157807304-de3abd4a-2e2c-4444-990b-f278b00c7e0c.png&data=04%7C01%7Csharmah%40algonquincollege.com%7C7d81c1fd195642cb5a8508da031fa19e%7Cec1bd9240a6a4aa9aa89c980316c0449%7C0%7C0%7C637825731636337209%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=mv2kX41V6vV8Zyyg2ASsQ0Gs23zxO1CWJ5RmxDzVUPw%3D&reserved=0>
So I think this is right. At the very least, it's what I used to plan the site I submitted to MarkBot, and that went through fine, hahah.
The only thing it's missing is the Homepage-Berries-Resources navigation that's on every page, but I wasn't sure how to draw that in without making a huge mess.
@sharmahAC<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FsharmahAC&data=04%7C01%7Csharmah%40algonquincollege.com%7C7d81c1fd195642cb5a8508da031fa19e%7Cec1bd9240a6a4aa9aa89c980316c0449%7C0%7C0%7C637825731636337209%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=8zcv0F3oE6LFiuyAITGHIhg8O3tTvSt6vIDxkfjvNqU%3D&reserved=0>
